### ISSUE TITLE
Rewriting paths leads to: unsupported protocol scheme \"\""

### ISSUE THREAD

**Issue by StarlessNights** [pb33f/wiretap#85]
Something seems to have broken in v0.1.0 release.

Routing requests to an RT server running on localhost:

```bash
$ podman run --rm --publish 9090:9090 --publish 9091:9091 --publish 9092:9092 --volume $PWD:/work:rw pb33f/wiretap:v0.1.0 --spec my_spec.yaml --url https://host.containers.internal

...

 INFO  wiretap is proxying all traffic to 'https://host.containers.internal'

 INFO  [wiretap] Re-writing path 'https://host.containers.internal/REST/2.0/rt' to '/REST/2.0/rt'
```

The info about the rewrite appears only after making a request, doesn't seem to matter which one.

And the response is:
```json
{
	"type": "https://pb33f.io/wiretap/error",
	"title": "Unable to call API",
	"status": 500,
	"detail": "Get \"/REST/2.0/rt\": unsupported protocol scheme \"\"",
	"payload": null
}
```

Same thing works fine in v0.0.49 and doesn't inform of any rewrites.

**Comment by daveshanley** [comment]
This is strange! will look into it.

**Comment by ikenna** [comment]
Yeah I am on 0.1.0 and I am getting same issue 
Client:
`% curl "http://localhost:9090/wiretap/giftshop/products"
{"type":"https://pb33f.io/wiretap/error","title":"Unable to call API","status":500,"detail":"Get \"/wiretap/giftshop/products\": unsupported protocol scheme \"\"","payload":null}%`

Wiretap:
`2024-01-22 12:42:54 INFO  [wiretap] handling API request url: /wiretap/giftshop/products
 INFO  [wiretap] Re-writing path 'https://api.pb33f.io/wiretap/giftshop/products' to '/wiretap/giftshop/products'
2024-01-22 12:42:54 INFO  [wiretap] request failed url: /wiretap/giftshop/products code: 500 error: Get "/wiretap/giftshop/products": unsupported protocol scheme ""
`

**Comment by jzyinq** [comment]
I had the same issue today while going through [quickstart](https://pb33f.io/wiretap/quickstart/).

**Comment by daveshanley** [comment]
Looking into this now.

**Comment by daveshanley** [comment]
This was a regression added by the HAR feature. It should be resolved now in `v0.1.1`

**Comment by ikenna** [comment]
Thank you @daveshanley ! 👍
